smarthashtagswithmap.py

```diff
@@ -42,14 +38,20 @@ def get_bounding_box(self, latitude_in_degrees, longitude_in_degrees, half_side_
         lat_max = lat + half_side_in_km / radius
         lon_min = lon - half_side_in_km / parallel_radius
         lon_max = lon + half_side_in_km / parallel_radius
-        rad2deg = math.degrees
```

Comment:
Why did you delete this line ?
I am sure by mistake

Reply:
Absolutely, sorry. Was going to import this with `as` but I actually liked your idea then, sorry 😉